Find a functionally equivalent code:android.app.Notification.lightenPayload:COMMENT
Method Modifier: public      final       hidden      
Comment:/**
 * Removes heavyweight parts of the Notification object for archival or for sending to
 * listeners when the full contents are not necessary.
 * @hide
 */

@@ -2,11 +2,15 @@
     tickerView = null;
     contentView = null;
     bigContentView = null;
+    headsUpContentView = null;
     largeIcon = null;
     if (extras != null) {
         extras.remove(Notification.EXTRA_LARGE_ICON);
         extras.remove(Notification.EXTRA_LARGE_ICON_BIG);
         extras.remove(Notification.EXTRA_PICTURE);
+        extras.remove(Notification.EXTRA_BIG_TEXT);
+        // Prevent light notifications from being rebuilt.
+        extras.remove(Builder.EXTRA_NEEDS_REBUILD);
     }
 }

